docs/overview/diagrams.md

```diff
@@ -121,6 +121,47 @@ classDiagram
 
+# AssociativeEither
+
+[![AssociativeCompose hierarchy][AssociativeEither-image]][AssociativeEither-link]
+
+```mermaid
+classDiagram
+  AssociativeEither~F<_>~ <|-- CommutativeEither~F<_>~
+  AssociativeEither~F<_>~ <|-- IdentityEither~F<_>~
+  class AssociativeEither~F<_>~{
+    Either[L, +*]
+    Exit[E, +*]
+    Fiber[E, +*]
+    Future[+*]
+    Schedule[R, E, +*]
+    Try[+*]
+    ZIO[R, E, +*]
+    ZLayer[R, E, +*]
+    ZManaged[R, E, +*]
+    ZStream[R, E, +*]
+    
+    () either[A, B](() => F[A], () => F[B]): F[Either[A, B]]
+  }
+  class CommutativeEither~F<_>~{
+    Future
+    ZIO[R, E, +*] 
+    ZSink[R, E, I, L, +*]
+    ZStream[R, E, +*]
+    Equal[-*]
+  }
+  class IdentityEither~F<_>~{
+    Option[+*]
+    Equal[-*]
+    Hash[+*]
+    Ord[+*]
```

Comment:
Please check the variance annotations on `Eqaul` and descendants (and also other, for double checking). `Equal` (and descendants) have _contra_-variant type parameter, so `-` (not `+`).
Thanks :pray:

Reply:
I checked it, Ord , Hash, Equal are contra-variant, so I changed to '-'